### PR TITLE
Feature/add employee name

### PR DESCRIPTION
**Pull request by niveditasa**
Added 'Nivedita' and 'Na Hyung' to the new employee array.

### CODE REVIEW

File: index.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-employees = []
+employees = ['Nivedita', 'Na Hyung']
```
Comment (schoinh) [inline index.py:1]: Hmmm question?

Comment (meganrm) [reply index.py:1]: I think this looks good

Comment (niveditasa) [reply index.py:1]: Answered

Comment (jxchen01) [reply index.py:1]: great!!

Comment (schoinh) [reply index.py:1]: Nivi I think you should add a comment to explain what this does

**Review by schoinh — CHANGES_REQUESTED** [review]

**Review by schoinh — APPROVED** [review]
Looks good!

**Review by meganrm — APPROVED** [review]

**Review by jxchen01 — APPROVED** [review]

**Pull request merged**